pyEpiabm/pyEpiabm/tests/test_func/test_sim_functional.py

```diff
@@ -139,6 +140,7 @@ def test_population_conservation(self, mock_mkdir, mock_output):
         mock_mkdir.assert_called_with(folder)
         nb_age_group = len(pe.Parameters.instance().age_proportions)
         mock_output_count = iter_num * nb_age_group * cell_count
+        print(mock_output_count)
```

Comment:
Spurious print statement - I guess that was in here for testing, but maybe should be taken out for the final version